README.md

```diff
@@ -4,53 +4,55 @@
 
```

Comment:
README is currently a work in progress and should be finished in the next week or two.